typings/test.ts

```diff
@@ -0,0 +1,8 @@
+import userEvent, {TargetElement} from '.'
+
+declare const element: TargetElement
+type NotVoid = string | number | boolean | object | null
+
+userEvent.type(element, 'foo', {delay: 1}) as Promise<void>
+// @ts-expect-error No delay returns void
```

Comment:
Awesome [usage of typescript](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#-ts-expect-error-comments) this was new to me many thanks! ❤️